CHANGES.md

```diff
@@ -9,6 +9,7 @@
 - Fix issue with recursive commponents [Taiga #9903](https://tree.taiga.io/project/penpot/issue/9903)
 - Fix missing methods reference on API Docs
 - Fix memory usage issue on file-gc asynchronous task (related to snapshots feature)
+- Fix issue on components with swap and ref [Taiga #8964](https://tree.taiga.io/project/penpot/issue/8964)
```

Comment:
2.4.2 is already released, this should go to 2.4.3

Reply:
fixed